pkg/rbac/rego.go

```diff
@@ -53,13 +53,23 @@ func init() {
 	}
 }
 
-func CheckAccess(ctx context.Context, action, resource string, roles []string) (bool, error) {
+func CheckAccess(ctx context.Context, action, resource, appSlug string, roles []string) (bool, error) {
+	allowRolePolicies := DefaultAllowRolePolicies
+	denyRolePolicies := DefaultDenyRolePolicies
+	if appSlug != "" {
```

Comment:
should this also include the support role?

Reply:
default includes it